### ISSUE TITLE
Code intelligence badge: _I want precise support!_ CTA

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#33795]
Following up from deficiencies of #33571 (PoC code intel badge) related to #31180 (communicate when precise code intel is available).

We should display the following information in the badge, which is not currently possible.

![image](https://user-images.githubusercontent.com/103087/162989433-9bfee155-14db-4240-846f-49e6ba423a6a.png)

We need to add a target for this link, which should lead to a form or a way for us to track the action via telemetry (in Cloud or on-prem).

**Comment by sourcegraph-bot-2** [comment]
Heads up @macraig - the "team/code-intelligence" label was applied to this issue.

**Comment by jjinnii** [comment]
@efritz To move this issue forward, for v1:

- Instead of a re-direct to the docs page, we'll communicate a confirmation for a CTA click within the badge. [Figma designs here](https://www.figma.com/file/p5rVseXjNvXRkvXCdttbWt/Semantic-badge?node-id=1009%3A14776) 
- Behavior: We should record one "request" per user per language. This means that we should maintain the "Received your request" state for users for any repo that would have the same language. Users can submit a request for _other_ languages, however.
- CTA clicks should be tracked via pings to be viewed in Looker. The goal is a more accurate view of language requests from our customers. (on an individual user basis versus a request from the whole customer). 
- It's a second lower priority to display this data in Amplitude for individual developer users, and can be done after v1. I can create an issue to track this. 

Let me know if this helps unblock you!

cc: @macraig

**Comment by efritz** [comment]
@jjinnii and @macraig - The bullet points above are implemented in #33869.
